app/src/main/java/at/aau/edu/wizards/MainActivity.kt

```diff
@@ -25,6 +30,11 @@ private const val SHARED_PREFERENCES_KEY = "SHARED_PREFS_USERDATA"
 
 class MainActivity : AppCompatActivity() {
 
+    val FINE_LOCATION = 144
```

Comment:
probably you see an warning here. Maybe it makes sense to use `android.Manifest.permission.ACCESS_FINE_LOCATION`